src/win/udp.c

```diff
@@ -204,11 +204,11 @@ static int uv__udp_maybe_bind(uv_udp_t* handle,
    * so we just return an error directly when UV_UDP_REUSEPORT is requested
    * for binding the socket. */
   if (flags & UV_UDP_REUSEPORT)
-    return ERROR_NOT_SUPPORTED;
+    return UV_ENOTSUP;
 
   if ((flags & UV_UDP_IPV6ONLY) && addr->sa_family != AF_INET6) {
     /* UV_UDP_IPV6ONLY is supported only for IPV6 sockets */
-    return ERROR_INVALID_PARAMETER;
+    return UV_EINVAL;
   }
```

Comment:
I think this change is unrelated and incorrect: uv__udp_maybe_bind should return a windows error.

Reply:
Roger that